src/sage/combinat/cluster_algebra_quiver/interact.py

```diff
@@ -25,8 +25,8 @@ def cluster_interact(self, fig_size=1, circular=True, kind='seed'):
 
     TESTS::
 
-        sage: S = ClusterSeed(['A',4])                                                  # optional - sage.graphs
-        sage: S.interact()   # indirect doctest                                         # optional - sage.graphs sage.symbolic
+        sage: S = ClusterSeed(['A',4])                                                  # optional - sage.graphs sage.modules
```

Comment:
Shouldn't it be `# needs sage.graphs sage.modules` ?

Reply:
That's now #35951